Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -60,6 +60,7 @@
     'AP_ICEngine',
     'AP_Frsky_Telem',
     'AP_FlashStorage',
+    'AP_UAVCAN',
```

Comment:
It's possible there is one or two here that aren't used in every vehicle, but certainly not 50%... Also, no other library is fully board dependent like this one is.

It shouldn't be included all the time because the majority of boards don't even have CAN. It's fine for now since AP_UAVCAN source code is protected by HAL_WITH_UAVCAN.